src/main/kotlin/com/ampnet/identityservice/controller/UserController.kt

```diff
@@ -1,15 +1,19 @@
 package com.ampnet.identityservice.controller
 
 import com.ampnet.identityservice.controller.pojo.request.EmailRequest
+import com.ampnet.identityservice.service.UserService
+import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-class UserController {
+class UserController(private val userService: UserService) {
 
```

Comment:
Add route: `@GetMapping("/user")` which returns User data: address: String and email: String?